docs/src/use/core-concepts/glossary.md

```diff
@@ -0,0 +1,396 @@
+For example, [@angular-eslint/eslint-plugin](https://www.npmjs.com/package/@angular-eslint/eslint-plugin) contains best practices for using the Angular framework.
+
+For more information, refer to [Configure Plugins](../configure/plugins).
+
+### Processor
+
+A part of a plugin that extracts JavaScript code from other kinds of files, then lets ESLint lint the JavaScript code.
+
+For example, [`eslint-plugin-markdown`](https://github.com/eslint/eslint-plugin-markdown) includes a processor that converts the text of <code>```</code> code blocks in Markdown files into code that can be linted.
+
+For more information on configuring processor, see [Plugins > Specify a Processor](../configure/plugins#specify-a-processor).
+
+## R
+
+### Report
+
+A collection of [violations](#violation) from a single ESLint run.
+
+When ESLint runs on source files, it will pass an [AST](#abstract-syntax-tree-ast) for each source file to each configured [rule](#rule).
+The collection of violations from each of the rules will be packaged together and passed to a [formatter](#formatter-linting) to be presented to the user.
+
+### Rule
+
+Code that checks an [AST](#abstract-syntax-tree-ast) for expected patterns. When a rule's expectation is not met, it creates a [violation](#violation).
+
+ESLint provides a large collection of rules that check for common JavaScript code issues.
+Many more rules may be loaded in by [plugins](#plugin).
+
+For an overview of rules provided, see [Core Concepts > Rules](../core-concepts#rules).
+
+## S
+
+### Selector
+
+Syntax describing how to search for [nodes](#node) within an [AST](#abstract-syntax-tree-ast).
+
+ESLint [rules](#rule) use [ESQuery](#esquery) selectors to find nodes that should be checked.
+
+### Severity
+
+What level of reporting a rule is configured to run, if at all.
+
+ESLint supports three levels of severity:
+
+* `"off"` (`0`): Do not run the rule.
+* `"warn"` (`1`): Run the rule, but don't exit with a non-zero status code based on its violations (excluding the [`--max-warnings` flag](../command-line-interface#--max-warnings))
+* `"error"` (`2`): Run the rule, and exit with a non-zero status code if it produces any violations
+
+For documentation on configuring rules, see [Configure Rules](../configure/rules).
+
+### Shareable Config (Configuration)
+
+A module that provides a predefined [config file](#config-file-configuration-file) configurations.
+
+Shareable configs can configure all the same information from config files, including [plugins](#plugin) and [rules](#rule).
+
+Shareable configs are often provided alongside [plugins](#plugin).
+Many plugins provide configs with names like _"recommended"_ that enable their suggested starting set of rules.
+For example, [`eslint-plugin-solid`](https://github.com/solidjs-community/eslint-plugin-solid) provides a shareable recommended config:
+
+```js
+import solid from "eslint-plugin-solid/configs/recommended";
```

Comment:
```suggestion
import js from "@eslint/js";
import solid from "eslint-plugin-solid/configs/recommended";
```

Since it's used in the config below.